### PR TITLE
Update release-notes.md

### PR DESCRIPTION
**Pull request by abukatar**
## Purpose of this pull request
 
Added RN for MQP v1.0.19
 
## Affected DevDocs pages
 
https://devdocs.magento.com/quality-patches/release-notes.html
 
whatsnew
Published [release notes](https://devdocs.magento.com/quality-patches/release-notes.html) for the 1.0.19 Magento Quality Patch (MQP) package release.

### CODE REVIEW

File: src/quality-patches/release-notes.md [hunk 1]
```diff
@@ -19,6 +19,23 @@ The [Magento Quality Patches](https://github.com/magento/quality-patches) packag
 See [Apply patches]({{ site.baseurl }}/guides/v2.4/comp-mgr/patching/mqp.html) for instructions on applying patches to your Magento projects.
 See [Patches available in MQP tool](https://support.magento.com/hc/en-us/sections/360010506631-Patches-available-in-MQP-tool-) for additional patch details.
 
+## v1.0.19
+
+-  **MDVA-33614** _(for Magento `2.4.1`)_-Fixes the issue where Page Builder throws the following error: "An error has occurred while initiating Page Builder. Please consult with your technical support contact."
+-  **MDVA-35356** _(for Magento `>=2.3.0 <2.4.3`)_-Fixes the issue with incorrect store credit return after partially invoiced order cancellation.
+-  **MDVA-33289** _(for Magento `>=2.4.0 <2.4.3`)_-Fixes the issue where raw JavaScript code is displayed in the billing address UI during checkout if Google Tag Manager is enabled.
+-  **MDVA-35982** _(for Magento `>=2.3.0 <2.4.3`)_-Fixes the issue where admin users restricted to a specific website cannot create a shipment for the order placed on same website.
+-  **MDVA-35155** _(for Magento `>=2.3.0 <2.3.6`)_-Fixes the issue where is impossible to buy a bundle product, if the option title was changed.
```
Comment (jeff-matthews) [inline src/quality-patches/release-notes.md:28]:
```suggestion
-  **MDVA-35155** _(for Magento `>=2.3.0 <2.3.6`)_-Fixes the issue where it is impossible to buy a bundle product if the option title was changed.
```

File: src/quality-patches/release-notes.md [hunk 2]
```diff
@@ -19,6 +19,23 @@ The [Magento Quality Patches](https://github.com/magento/quality-patches) packag
...
+-  **MDVA-35910** _(for Magento `>=2.4.1 <2.4.3`)_-Fixes the issue where it is impossible to create a new customer account after disabling the Login as Customer functionality.
+-  **MDVA-34474** _(for Magento `>=2.3.0 <2.4.3`)_-Fixes the issue with adding a product to the requisition list using API.
```
Comment (jeff-matthews) [inline src/quality-patches/release-notes.md:30]:
```suggestion
-  **MDVA-34474** _(for Magento `>=2.3.0 <2.4.3`)_-Fixes the issue with adding a product to the requisition list using the API.
```

**Review by jeff-matthews — CHANGES_REQUESTED** [review]
Minor editorial suggestions

**Comment by jeff-matthews** [comment]
@magento import code to magento-devdocs/devdocs

**Comment by magento-engcom-team** [comment]
@jeff-matthews the branch with code successfully imported into`magento-devdocs/devdocs` repository. Branch name: `imported-magento-devdocs-8846`.

**Pull request merged**